### PR TITLE
New PR for FS preview

### PR DESCRIPTION
**Pull request by jzila**
@keybase/react-hackers

This has had a round of feedback but we haven't gotten final approval.

### CODE REVIEW

File: shared/constants/fs.js [hunk 1]
```diff
@@ -182,6 +182,21 @@ const itemStylesTeamTlf = memoize((teamName: string) => ({
...
+  const gib = mib * kib
+  const tib = gib * kib
+
+  if (!meta) return ''
+  const size = meta.size
+  if (size >= tib) return `${Math.round(size / tib)}TB`
+  if (size >= gib) return `${Math.round(size / gib)}GB`
+  if (size >= mib) return `${Math.round(size / mib)}MB`
+  if (size >= kib) return `${Math.round(size / kib)}KB`
+  return '' + size
```
Comment (jzila) [inline shared/constants/fs.js]: This should be `` `${size}B` ``

Comment (songgao) [reply shared/constants/fs.js]: I think there should also be a space before the unit? The popup design has space but not in the file info screen.

Comment (jzila) [reply shared/constants/fs.js]: Yup, sure.

**Comment by jzila** [comment]
Sigh, timezone issue in storyshots.

**Pull request merged**